archetypes/blog.md

```diff
@@ -5,7 +5,7 @@ date: "{{ .Date }}"
 draft: false
 image: img/blog/
 photo_credit: "Add Photo Credits Here"
-author: Your Name
+authors: ['Your Name', 'Co-Authors Name']
```

Comment:
Hugo requires that taxonomies are plural, so each blog post required that `author` be replaced with `authors`. The author's name also has to be enclosed with brackets, even for single names. I decided to go ahead and include an update to the archetype file.